src/main/resources/reference.conf

```diff
@@ -7,6 +7,8 @@ jicofo {
     max-bridge-packet-rate = 50000
     // The assumed average packet rate per participant.
     average-participant-packet-rate-pps = 500
+    // The assumed average load amount per participant
+    average-participant-load = 0.01
```

Comment:
This should probably be called "stress" rather than "load"? And is 0.01 too high? (100 participants per bridge feels low).

Reply:
> This should probably be called "stress" rather than "load"?

Sure

> And is 0.01 too high? (100 participants per bridge feels low).

Yeah this was a bit of a placeholder to discuss.